docs/README.md

```diff
@@ -74,6 +74,20 @@ or the shortcut
 cz c
 ```
 
+### Integrating with Pre-commit
+Commitizen can lint your commit message for you with `cz check`.
+You can integrate this in your [pre-commit](https://pre-commit.com/) config with:
+```yaml
```

Comment:
Adding this config file is not enough for `pre-commit`, I'd suggest add the `pre-commit install --hook-type commit-msg` command

Reply:
Hmm. This already assumes that the dev has run `pre-commit` before tho. When I was trying this out for myself I just added that snippet to my config and reran pre-commit. I think the `stages: [commit-msg]` is enough.

Reply:
Hey @Lee-W, just to follow up, you're right. It needs the `pre-commit install --hook-type commit-msg` command.

Reply:
Ah, I miss this part when I merged this PR.  No worries. I'll add it in another PR.